<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mashStepEditor</class>
 <widget class="QDialog" name="mashStepEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mash Step Editor</string>
  </property>
  <property name="configSection" stdset="0">
   <string notr="true">mashStepEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMashStepDetails">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="BtTemperatureLabel" name="label_stepTemp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Target temp.</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_stepTemp</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="BtTemperatureEdit" name="lineEdit_stepTemp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Target temp. of this step</string>
         </property>
         <property name="editField" stdset="0">
          <string notr="true">stepTemp_c</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="BtTemperatureLabel" name="label_endTemp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>End temp.</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_endTemp</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="BtTemperatureEdit" name="lineEdit_endTemp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Final temp. of this step</string>
         </property>
         <property name="editField" stdset="0">
          <string notr="true">endTemp_c</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="BtTimeLabel" name="label_stepTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_stepTime</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="BtTimeEdit" name="lineEdit_stepTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Time to conduct the step</string>
         </property>
         <property name="editField" stdset="0">
          <string notr="true">stepTime_min</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="BtTimeLabel" name="label_rampTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Temp. lag time</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_rampTime</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="BtTimeEdit" name="lineEdit_rampTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Lag time</string>
         </property>
         <property name="editField" stdset="0">
          <string notr="true">rampTime_min</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_type</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_type">
         <property name="toolTip">
          <string>Type</string>
         </property>
         <item>
          <property name="text">
           <string>Infusion</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Temperature</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Decoction</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMashStepProperties">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Infusion</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="BtVolumeLabel" name="label_infuseAmount">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="text">
             <string>Infuse amount</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_infuseAmount</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="BtVolumeEdit" name="lineEdit_infuseAmount">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Amount of water to infuse</string>
            </property>
            <property name="editField" stdset="0">
             <string notr="true">infuseAmount_l</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="BtTemperatureEdit" name="lineEdit_infuseTemp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Temperature of infusion water</string>
            </property>
            <property name="editField" stdset="0">
             <string notr="true">infuseTemp_c</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="BtTemperatureLabel" name="label_infuseTemp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="text">
             <string>Infuse temp.</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_infuseTemp</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Decoction</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="BtVolumeLabel" name="label_decoctionAmount">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="text">
             <string>Decoction amount</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_decoctionAmount</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="BtVolumeEdit" name="lineEdit_decoctionAmount">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Amount of mash to decoct</string>
            </property>
            <property name="editField" stdset="0">
             <string notr="true">decoctionAmount_l</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMashStepInfo">
      <attribute name="title">
       <string>Info</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtTemperatureEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTimeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTimeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mashStepEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mashStepEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_stepTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_stepTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_infuseAmount</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_infuseAmount</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_infuseTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_infuseTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_decoctionAmount</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_decoctionAmount</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_endTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_endTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_stepTime</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_stepTime</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_rampTime</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_rampTime</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
